docs/docs/evaluation/configuration-reference.md

```diff
@@ -0,0 +1,227 @@
+# Configuration Reference
+
+This page contains reference documentation for the configuration options for the MongoDB Chatbot Evaluation CLI.
+
+A Evaluation CLI config files is a CommonJS file that exports a `ConfigConstructor` function as its default export.
+
+For an example of setting up a configuration file, refer to the [Configuration](./index.md) documentation.
+
+To set up a configuration file, you must first install the following packages:
+
+```bash
+npm install mongodb-chatbot-evaluation
+```
```

Comment:
Should we link out like on the other reference page? e.g. " You need to have the Evaluation CLI installed to configure it. See the installation docs."?